site/content/en/docs/Guides/_index.html

```diff
@@ -1,9 +0,0 @@
----
-title: "Guides"
```

Comment:
Bring this file back. Docsy looks like it wants this as a standard, an index page with a brief description of the overall section.

Reply:
Deleted Guide folder itself.